docs/source/markdown/podman-run.1.md

```diff
@@ -89,14 +89,15 @@ Set the cgroup namespace mode for the container.
 
 If the host uses cgroups v1, the default is set to **host**.  On cgroups v2, the default is **private**.
 
-**--cgroups**=**enabled**|**disabled**|**no-conmon**
+**--cgroups**=**enabled**|**disabled**|**no-conmon**|**conmon-delegated**
 
 Determines whether the container will create CGroups.
 
 Default is **enabled**. The **disabled** option will force the container
 to not create CGroups, and thus conflicts with CGroup options
 (**--cgroupns** and **--cgroup-parent**).
 The **no-conmon** option disables a new CGroup only for the **conmon** process.
+The **conmon-delegated** option reuses the current cgroup for both the conmon and the container payload, it works only on cgroup v2.
```

Comment:
If it's also specific to systemd cgroups (looks like it is) we should also mention that

Reply:
not sure it is so specific to systemd.  It can be used also without systemd, even if it is less useful

Reply:
this will also break remote.  We probably need a check to not permit the option being used as remote as it changes the cgroup of the Podman process itself